bitwarden_license/bit-web/src/app/secrets-manager/layout/navigation.component.html

```diff
@@ -5,6 +5,6 @@
 <org-switcher [filter]="orgFilter"></org-switcher>
 <bit-nav-item icon="bwi-collection" text="Projects" route="projects"></bit-nav-item>
 <bit-nav-item icon="bwi-key" text="Secrets" route="secrets"></bit-nav-item>
-<bit-nav-item icon="bwi-wrench" text="Service Accounts" route="service-accounts"></bit-nav-item>
+<bit-nav-item icon="bwi-wrench" text="Service accounts" route="service-accounts"></bit-nav-item>
```

Comment:
Whops we should localize these

```suggestion
<bit-nav-item icon="bwi-wrench" [text]="'serviceAccount' | i18n" route="service-accounts"></bit-nav-item>
```